README.md

```diff
@@ -21,10 +21,10 @@ npm install -g contentful-migration-cli
 contentful-migration --space-id <space id to use> <path to migration description file>
 ```
 
-In your migration description file, export a function that accepts the `migration` object as its argument. For example:
+In your migration description file, export a function that accepts the `migration` object and the `config` for the migrations as its arguments. For example:
 
 ```javascript
-module.exports = function (migration) {
+module.exports = function (migration, config) {
```

Comment:
I think `config` is kind of a bad name for this.  
Can't come up with a much better one though.  
Maybe something like `toolkit` or something in that direction?

Reply:
To clarify, I think `config` does make sense for the token and space, but `makeRequest` doesn't really fit in.  
We could also maybe split those things apart?